File: fellowGenius-angular/src/app/model/tutorAvailabilitySchedule.ts
import { scheduleData } from './scheduleData';

export class tutorAvailabilitySchedule {
	public tid: number;
	public fullName: string;
	public allAvailabilitySchedule: scheduleData[];
	public allMeetingsSchedule: scheduleData[];
	public availabilityStatus: string;
	public isAvailable: string;
}

File: fellowGenius-spring/src/main/java/fG/Entity/TutorAvailabilitySchedule.java
package fG.Entity;

import lombok.Data;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.Date;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Lob;


@Entity
@Data
public class TutorAvailabilitySchedule implements Serializable {
	private static final long serialVersionUID = 1L;
	@Id
	Integer tid;
	@Lob
	@Column(columnDefinition="BLOB")
	ArrayList<String> allAvailabilitySchedule;
	String fullName;
	String isAvailable;
	Date noScheduleNotificationTime;

	@Override
	public String toString() {
		return "TutorAvailabilitySchedule [tid=" + tid + ", allAvailabilitySchedule=" + allAvailabilitySchedule
				+ ", fullName=" + fullName + ", isAvailable=" + isAvailable + ", noScheduleNotificationTime="
				+ noScheduleNotificationTime + "]";
	}

	
}
